worker/instancemutater/doc.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package Instancemutater contains brokers for environ and machine containers.
+// The machine container is a worker that watches all the model machines that
```

Comment:
The ContainerWorker runs on a machine and watches for containers to be created on it. 

The EnvironWorker watches for machines in the model to be created.